ChatQnA/tests/test_compose_on_xeon.sh

```diff
@@ -178,13 +178,26 @@ function stop_docker() {
 function main() {
 
     stop_docker
+    # install perfspect
+    wget -qO- https://github.com/intel/PerfSpect/releases/latest/download/perfspect.tgz | tar xvz
+
+    # get used disk
+    list="$(./perfspect/perfspect report --filesystem 2>&1)"
+    list=$( for i in ` echo  $list `; do [[ $i =~ json ]] && echo $i ; done )
+    used_disk_before="$(python parse_perfspect.py $list 2>&1)"
```

Comment:
Line 181 to line 186 could be part of `parse_perfspect.py`, since it's common used for all examples.